lightning/src/ln/chan_utils.rs

```diff
@@ -116,19 +116,27 @@ pub const HTLC_SUCCESS_INPUT_P2A_ANCHOR_WITNESS_WEIGHT: u64 = 324;
 /// The size of the 2-of-2 multisig script
 const MULTISIG_SCRIPT_SIZE: u64 = 1 + // OP_2
 	1 + // data len
-	33 + // pubkey1
+	bitcoin::secp256k1::constants::PUBLIC_KEY_SIZE as u64 + // pubkey1
 	1 + // data len
-	33 + // pubkey2
+	bitcoin::secp256k1::constants::PUBLIC_KEY_SIZE as u64 + // pubkey2
```

Comment:
Honestly I find these constants kinda unreadable - a public key's "size" really should be 64 bytes (ish) cause its two coordinates. The fact that `PUBLIC_KEY_SIZE` refers to the serialized size when serialized in *compressed* form is very surprising. Same goes for `MAX_SIGNATURE_SIZE`, which confusingly refers to the maximum size of a signature *we generate* (depending on settings, often its not even that), not the actual maximum signature size.

Reply:
> Honestly I find these constants kinda unreadable - a public key's "size" really should be 64 bytes (ish) cause its two coordinates. The fact that `PUBLIC_KEY_SIZE` refers to the serialized size when serialized in _compressed_ form is very surprising.

Hmm... there is also `UNCOMPRESSED_PUBLIC_KEY_SIZE`, so seems this complaint is more directed at `secp256k1` crate not using a similar prefix for `PUBLIC_KEY_SIZE`?

> Same goes for `MAX_SIGNATURE_SIZE`, which confusingly refers to the maximum size of a signature _we generate_ (depending on settings, often its not even that), not the actual maximum signature size.

It's the maximum that `secp256k1` will generate, though, which is what we use.